redfish-core/lib/log_services.hpp

```diff
@@ -718,6 +767,14 @@ inline void
                     crow::utility::getDateTime(timestamp);
                 asyncResp->res.jsonValue["AdditionalDataSizeBytes"] = size;
 
+                if (!clientId.empty())
+                {
```

Comment:
Don't need an odata.id here? 
https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/47083/3/redfish-core/lib/account_service.hpp

Reply:
Not sure if it is needed. Validator passed without the odata.id actually.

Reply:
https://sys-openbmc-dev-jenkins.swg-devops.com/job/OpenBMC-Dev/job/test/job/continuous-integration/job/ebmc/job/Redfish-Service-Validator/440/artifact/logs/ConformanceHtmlLog_11_08_2021_082702.html

Reply:
![Screenshot from 2021-11-08 15-38-07](https://user-images.githubusercontent.com/47350623/140723292-5b9ae7f5-92a5-497a-b2ef-99bd747a79e9.png)